### DISCUSSION TITLE
[Script request]  Samba Share in host

### DISCUSSION THREAD

**Discussion by thor9898** [community-scripts/ProxmoxVE#997] [Request script]
### Application Name

Samba Share

### Website

https://www.samba.org/

### Description

First, I want to express my gratitude to everyone who has contributed to this project, especially the late tteck, whose work has been incredibly helpful and inspiring. It has motivated me to learn and improve significantly.

I would like to propose a feature request: a script to simplify the creation of an SMB share on the host. Ideally, this script could also handle user creation and manage the required permissions. This functionality might fit well under the Proxmox VE Tools category.

Unfortunately, I lack the technical expertise to contribute to this feature myself, but I hope this idea can be considered and implemented by the community.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by HearthCore** [comment]
This is somewhat already available with a combination of:
- Cockpit LXC via Helper Scripts https://community-scripts.github.io/ProxmoxVE/scripts?id=cockpit
- 45Drives Cockpit Plugin for managing Samba and NFS shares https://github.com/45Drives/cockpit-file-sharing
- 45Drives Cockpit Plugin for User and Group Management https://github.com/45Drives/cockpit-identities

**Reply by Sideflare** [reply]
I too would like to see a simple lxc containerized samba. After tossing cockpit out i ended up duplicating a cloudflared lxc and having claude delete cloudflare and map my zfs datasets to it and whip up a smb.conf.   feel like this would be an easy community script that should be present already. turnkey file server lxc is not as cool as community script ones with the preconfigured passwordless login.  on that note, I would also love it if there was just a blank lxc creator!
